torch_xla/runtime.py

```diff
@@ -139,7 +139,7 @@ def global_device_count() -> int:
 
 @requires_pjrt
 def world_size() -> int:
-  """Returns the total number of configured logical devices."""
```

Comment:
This is still the number of configured logical devices. For SPMD, that number happens to be equal to the number of processes, since there's one logical device per process.

Reply:
Per our discussion. let me know what you think of the new comment.